src/main/java/com/snowflake/kafka/connector/internal/ingestsdk/KcStreamingIngestClient.java

```diff
@@ -60,14 +60,17 @@ public KcStreamingIngestClient(SnowflakeStreamingIngestClient client) {
    * <p>Any exceptions will be passed up, a sfexception will be converted to a connectexception
    *
    * @param streamingClientProps the properties for the client
+   * @param parameterOverrides Helps to override any default parameters in streaming client
    * @param clientName the client name to uniquely identify the client
    */
-  protected KcStreamingIngestClient(Properties streamingClientProps, String clientName) {
+  protected KcStreamingIngestClient(
+      Properties streamingClientProps, Map<String, Object> parameterOverrides, String clientName) {
```

Comment:
could add the streaming version number to parquet/arrow enum or map here or in the ingestSdkProvider. (I'm assuming its like { (0 -> arrow), (1 -> parquet) }? If not ignore this comment lol)

Reply:
never mind i see the enum at the bottom